src/main/java/org/broadinstitute/hellbender/tools/walkers/sv/SVCleanPt1a.java

```diff
@@ -0,0 +1,439 @@
+ * </pre>
+ *
+ * <h3>Cleaning Steps</h3>
+ * <ol>
+ *     <li>
+ *         Adds new FILTER and INFO tags to header.
+ *     </li>
+ *     <li>
+ *         TODO
+ *     </li>
+ * </ol>
+ */
+@CommandLineProgramProperties(
+        summary = "Clean and format structural variant VCFs per Step 1a",
+        oneLineSummary = "Clean and format structural variant VCFs per Step 1a",
+        programGroup = StructuralVariantDiscoveryProgramGroup.class
+)
+@BetaFeature
+@DocumentedFeature
+public final class SVCleanPt1a extends VariantWalker {
+    public static final String PED_FILE_LONG_NAME = "ped-file";
+    public static final String CHRX_LONG_NAME = "chrX";
+    public static final String CHRY_LONG_NAME = "chrY";
+    public static final String FAIL_LIST_LONG_NAME = "fail-list";
+    public static final String PASS_LIST_LONG_NAME = "pass-list";
+    public static final String OUTPUT_SAMPLES_LIST_LONG_NAME = "sample-list";
+    public static final String OUTPUT_REVISED_EVENTS_LIST_LONG_NAME = "revised-list";
+
+    @Argument(
+            fullName = PED_FILE_LONG_NAME,
+            doc = "Sample PED file"
+    )
+    private GATKPath pedFile;
+
+    @Argument(
+            fullName = CHRX_LONG_NAME,
+            doc = "chrX column name"
+    )
+    private String chrX;
+
+    @Argument(
+            fullName = CHRY_LONG_NAME,
+            doc = "chrY column name"
+    )
+    private String chrY;
+
+    @Argument(
+            fullName = FAIL_LIST_LONG_NAME,
+            doc = "List of complex variants failing the background test"
+    )
+    private GATKPath failList;
+
+    @Argument(
+            fullName = PASS_LIST_LONG_NAME,
+            doc = "List of complex variants passing both sides"
+    )
+    private GATKPath passList;
+
+    @Argument(
+            fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME,
+            shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME,
+            doc = "Output VCF name"
+    )
+    private GATKPath outputVcf;
+
+    @Argument(
+            fullName = OUTPUT_SAMPLES_LIST_LONG_NAME,
+            doc="Output list of samples"
+    )
+    private GATKPath outputSamplesList;
+
+    @Argument(
+            fullName = OUTPUT_REVISED_EVENTS_LIST_LONG_NAME,
+            doc="Output list of revised genotyped events"
+    )
+    private GATKPath outputRevisedEventsList;
+
+    private VariantContextWriter vcfWriter;
+    private BufferedWriter samplesWriter;
+    private BufferedWriter revisedEventsWriter;
+
+    private Map<String, Integer> sampleSexMap;
+    private Set<String> failSet;
+    private Set<String> passSet;
+    private Set<String> writtenRevisedEvents = new HashSet<>();
+
+    private static final int MIN_ALLOSOME_EVENT_SIZE = 5000;
+
+
+    @Override
+    public void onTraversalStart() {
+        // Read supporting files
+        sampleSexMap = readPedFile(pedFile);
+        failSet = readLastColumn(failList);
+        passSet = readLastColumn(passList);
```

Comment:
You could use `TableReader` here. It would require you to add header lines to the lists in the WDL, which would be okay too. See `TableUtils.reader()` and look at some implementations to see examples.